.github/workflows/RunTests.yml

```diff
@@ -0,0 +1,30 @@
+# name: Run Tests
+
+# on:
+#   push:
+#     branches:
+#       - main
+#       - dev
+#   pull_request:
+#     types: [opened, synchronize, reopened]
+
+# jobs:
+#   test:
+#     runs-on: ubuntu-latest
+
+#     steps:
+#       - name: Checkout repository
+#         uses: actions/checkout@v2
+
+#       - name: Set up Node.js
+#         uses: actions/setup-node@v3
+#         with:
+#           node-version: '18'
```

Comment:
mesmo estando comentado, se isto for para ser utilizado no futuro, a versão do node deve ser igual à do dockerfile.